packages/ui/src/council/components/election/CandidateVote/CandidateVote.tsx

```diff
@@ -58,7 +59,8 @@ export const CandidateVote = ({
   const roundedPercentage = totalStake.gt(BN_ZERO) ? sumOfAllStakes.muln(100).divRound(totalStake).toNumber() : 0
   const userVoted = myVotes.length > 0
   const allVotesRevealed = myVotes.every((vote) => vote.voteFor)
-
+  const tooltipLabel = "Vote for your own membership with the same account as used for council candidacy creation counts towards the progress. Please note, this will in a voting lock applied to this account, which may only be withdrawn in the end of the council period if you win the election and released immediately if your candidacy gets outvoted by others. Voting locks are non-rivalrous."
```

Comment:
1. This is a bit unclear: 'counts towards the progress'
2. 'this will [result] in a voting lock'

Reply:
I fixed the typo and added an issue to reformulate the tooltip later on: #4153

Reply:
Addressed in #4155